### DISCUSSION TITLE
How to setup a smart host in /opt/postal/config/postal.yml

### DISCUSSION THREAD

**Discussion by rancohen-dreamvps** [postalserver/postal#3174] [Help with using Postal]
Hi
I need to know how to use a customized SMTP server as a smart host like amazon SES or any Other 3rd party SMTP
what needs to be configured exactly in /opt/postal/config/postal.yml?
From what I see there are smtp: and smtp_server: but what is the purpose of each?

Thanks

**Comment by willpower232** [comment]
smtp_server is for configuring Postal itself, what port should it use and what certificate should be used if TLS is enabled.

smtp is for configuring Postals ability to send its own emails, i.e. password resets.

If you get into the web UI, you will see you can add Organizations, Servers, domains, and credentials which I think is what you're looking for.
